view/frontend/web/js/view/payment/omise.js

```diff
@@ -15,8 +15,9 @@ define(
             'offsite-alipay',
             'offsite-installment',
             'offsite-truemoney',
-            'offsite-pointsciti',
-            'offline-tesco'
+            'offline-tesco',
+            'offline-paynow',
```

Comment:
Are we always including all of these even if the payment methods are disabled?

Reply:
yes.

Reply:
That doesn't seem right, or a good idea?

Reply:
If someone enabled for example just credit card payments, then all the JS code for all payment methods is going to be downloaded. This probably isn't ideal - especially on slow mobile connections

Reply:
it is not ideally.. but actually i think it won't be downloaded if in admin panel payment is set to `off`